packages/wonder-blocks-dropdown/components/action-menu.test.js

```diff
@@ -106,7 +107,7 @@ describe("ActionMenu", () => {
                 <OptionItem label="Toggle B" value="toggle_b" />
             </ActionMenu>,
         );
-        menu.setState({open: true});
+        menu.find(ClickableBehavior).simulate("click");
```

Comment:
This is great! I find it better to simulate stuff instead of manipulating state directly :)

Reply:
thanks! yes that’s what I was trying to achieve :)

Reply:
Does simulating events on `ActionMenu` itself not work?

Reply:
For some reason it doesn’t on ActionMenu

Reply:
I wonder if bubble/capture doesn't work with enzyme. 🤔